README.md

```diff
@@ -91,15 +91,15 @@ driven through the OLM.
 
 # Getting started
 
-A package of operator code is called a charm. You will use `charmcraft`
+A package of operator code is called a charmed operator. You will use `charmcraft`
```

Comment:
Can we add ` or "charm"` (just here where we're defining things, not in other places)? The term "charm" is still widely used, and it would be nice to acknowledge that, even if we have "charmed operator" first and primary.

Reply:
Sure, for terminology introduction we've been using "Charmed operator, packaged as a “charm”"